obs_status.yml

```diff
@@ -0,0 +1,1995 @@
+  mp_starcat_time: 2020:026:14:02:58.573
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:045:02:16:37.894
+  status: 1
+- agasc_id:
+  - 110493968
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:055:11:57:17.800
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:057:21:03:53.096
+  status: 1
+- agasc_id:
+  - 111028544
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:059:19:29:07.754
+  status: 1
+- agasc_id:
+  - 187829568
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:060:11:59:00.012
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:063:06:09:44.589
+  status: 1
+- agasc_id:
+  - 33690776
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:063:09:21:38.457
+  status: 1
+- agasc_id:
+  - 798760200
+  comments: Never acquired; set mag 11.2 +/- 0.1
+  mp_starcat_time: 2020:083:13:35:57.577
+  status: 1
+- agasc_id:
+  - 553396016
+  comments: High Background
+  mp_starcat_time: 2020:096:13:24:30.943
+  status: 1
+- agasc_id:
+  - 891159184
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2020:139:21:23:16.181
+  status: 1
+- agasc_id:
+  - 891159184
+  comments: Almost never tracked. Set mag to MAXMAG
+  mp_starcat_time: 2020:140:00:46:15.767
+  status: 1
+- agasc_id:
+  - 694561568
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:228:13:19:56.547
+  status: 1
+- agasc_id:
+  - 396501520
+  comments: Never acquired; set mag to MAXMAG
+  mp_starcat_time: 2020:240:12:46:53.718
+  status: 1
+- agasc_id:
+  - 396501520
+  comments: Never acquired; set mag to MAXMAG
+  mp_starcat_time: 2020:243:11:17:18.150
+  status: 1
+- agasc_id:
+  - 42864912
+  comments: Faint star. Set mag to 12.6 +- 0.3
+  mp_starcat_time: 2020:259:07:03:15.474
+  status: 1
+- agasc_id:
+  - 77073552
+  comments: Faint star. Never acquired
+  mp_starcat_time: 2020:261:23:04:56.959
+  status: 1
+- agasc_id:
+  - 405276920
+  comments: Never acquired; set mag to MAXMAG; tracking interval is spurious
+  mp_starcat_time: 2020:266:08:19:47.944
+  status: 1
+- agasc_id:
+  - 690625640
+  comments: Never acquired; set mag to MAXMAG
+  mp_starcat_time: 2020:267:05:49:07.272
+  status: 1
+- agasc_id:
+  - 686557584
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:272:02:24:05.125
+  status: 1
+- agasc_id:
+  - 686557784
+  comments: Never acquired. Set mag to MAXMAG
+  mp_starcat_time: 2020:272:02:24:05.125
+  status: 1
+- agasc_id:
+  - 80616784
+  comments: Tracking affected by imposter
+  mp_starcat_time: 2020:280:11:16:08.609
+  status: 1
+- agasc_id:
+  - 82972168
+  comments: Acquired imposter. Out of view
+  mp_starcat_time: 2020:304:03:02:34.117
+  status: 1
+- agasc_id:
+  - 171724672
+  comments: Mag 7.6 spoiler 13 arcsec away
```

Comment:
I think this might be a high proper motion star for which we have no proper motion in the agasc:
http://simbad.u-strasbg.fr/simbad/sim-id?Ident=%40795861&Name=HD%20%2037124&submit=submit ?

Reply:
I added this to the bad table as a high proper motion star